ACI Worker pull request

towelenee opened the pull request.

Commit:
Add support for different runtimes

Commit:
Add worker concurency flag

Commit:
fix ingestion

Commit:
Clone worker to aci_worker

Dockerfile with params

Commit:
SoftTimeLimitExceeded import r

1. SoftTimeLimitExceeded fixed
2. removed commented docker code

Commit:
WIP cloud-hunky 

1. fixed PyYaml version for building
2. added cloud-hunky calls

Commit:
WIP ACI

Commit:
WIP aci

Commit:
WIP refactor

Commit:
WIP 2 refactoring

Commit:
Tested Pre-final

Commit:
WIP Docker wrap

Commit:
fixed celery's decorator import bug

Commit:
fixed Concurrency cleaning files bug

Commit:
Fixed "No code submission" bug

Commit:
package logging

Commit:
cleaning + full logging with clear time

Commit:
deploy.sh file

towelenee commented on .env_sample:
```diff
@@ -1,5 +1,35 @@
+AZURE_AUTH_LOCATION=/root/.azure/azure_creds
+
+WORKER=celerymain
+WORKER_CONCURRENCY=2
```
WORKER_CONCURRENCY 2 doesn't work in a classical worker?

igormusinov replied:
WORKER_CONCURRENCY doesn't work in classic due to garbage cleaning (I know how to fix this https://github.com/towelenee/competitions-v1-compute-worker/blob/master/worker.py#L289-L294). But there was one big problem. All workers of one celery process have to use one common Azure File Share Storage, that means that they could read/write/execute each other's files. Now I create "N" single celery workers (= N dockers) every with independent AFS Shares ("subdirectory" of one storage). 

There were 2 alternatives:
1. Give permissions only to apropriate folder and use only root user. But this looks cumbersome and potentially unsafe. 
2. Use Secrets to download and upload S3 data from inside containers, but  I'm not sure that it is a good idea. There is a little chance that it is not safe. On the other hand it  can save 1-9 minutes per submission due acceleration of data transfer.

igormusinov replied:
Was
![image](https://user-images.githubusercontent.com/11311889/62381201-a19f5e80-b553-11e9-9856-c57b0a1c6504.png)

Now
![image](https://user-images.githubusercontent.com/11311889/62382181-b7158800-b555-11e9-8f26-b721ba1b758c.png)

towelenee commented on Dockerfile:
```diff
@@ -1,21 +1,25 @@
-FROM python:2.7
+FROM python:3
```
python:3-slim is much lighter

igormusinov replied:
Not so important

Commit:
Fix bug with AFS files

1. ingestion removed
2. Fixed bug with open files that wasn't closed when removing AFS files

towelenee reviewed (commented):
Okish, mb some place for refactoring

Commit:
Ansible-deploy

Commit:
Ansible-deploy

Commit:
Ansible deploy with pulling

Commit:
Add ACI params

- add emacs for inner debugging
- removed trash files

Commit:
Add some common configs

Commit:
Ansible with tags

Commit:
README for ACI-workers

Commit:
Update README.md

Commit:
[Ref] deploy 

1. add file with list of workers with stage type
2. docker deploy fixes (with list + stage configs)

Commit:
boost + tag stage.type

Commit:
FIX Mismount

Commit:
steps

Commit:
gpu_count_bugfix

Commit:
fix

Commit:
fix

Commit:
lets use submission_id no task_id

Commit:
Merge pull request #1 from marts2007/master

steps variable for "Mount to saved weights for user visibility" task
